ui/components/UserPreferences/style.js

```diff
@@ -7,7 +7,7 @@ const styles = (theme) => ({
     backgroundColor: theme.palette.primary,
     color: theme.palette.secondary.text,
     '@media (max-width: 368px)': {
-      width: '180px',
+      width: '100%',
```

Comment:
😬 px, thanks for fixing it, good work 👏

Reply:
@Yashsharma1911 should i change the text size